core_perception/naive_motion_predict/launch/naive_motion_predict.launch

```diff
@@ -2,7 +2,7 @@
 <launch>
     <arg name="interval_sec" default="0.1"/>
     <arg name="num_prediction" default="10"/>
-    <arg name="sensor_height" default="2.0"/>
+    <arg name="sensor_height_calibration_params_file" default="$(find vehicle_calibration_dir)/naive_motion_predict/calibration.yaml"/>
```

Comment:
For the default value you should use the local package. So $(find naive_motion_predict). There is no vehicle_calibration_dir package to locate so this would fail if run from source.